tests/Test/src/basic/test_basic_logsumexp.birch

```diff
@@ -0,0 +1,149 @@
+
+/*
+ * Test log-sum-exp implementations in `log_sum_exp` and
+ * `resample_reduce`.
+ */
+program test_logsumexp() {
```

Comment:
Should be `test_basic_logsumexp` to be picked up and run by the `grep` in `smoke.sh` and `test.sh`. Or I'd even suggest `test_basic_log_sum_exp` here to be consistent with the name of the function.

Reply:
Similar throughout, `logsumexp` -> `log_sum_exp`.

Reply:
Ah I had noticed this probably as well and thought I had pushed a commit that fixes the name. It seems I did not :smile:

Reply:
Oh, actually I think I renamed it to `test_basic_logsumexp` in https://github.com/lawmurray/Birch/pull/18/commits/bc6d13ac0e2fa0fc97f0da926f72f5c6a491e493, so maybe you were looking at an older commit? In any case, I'll rename `logsumexp` to `log_sum_exp`.

Reply:
I renamed all occurrences of `logsumexp` (also in the filename) to `log_sum_exp`.